src/briefcase/commands/create.py

```diff
@@ -379,6 +379,40 @@ def install_app_support_package(self, app: BaseConfig):
             print()
             raise InvalidSupportPackage(support_package_url)
 
+    def build_env(self, app: BaseConfig):
+        """
+        Return an environment in which to build the dependencies for the app.
+        This should include the path to Python.h in CPPFLAGS, and libpython
+        and its path in LDFLAGS.
+
+        :param app: The config object for the app
+        """
+        env = dict(os.environ)
+        if self.platform == "android":
+            # sysconfig.get_config_var("INCLUDEPY")
+            includepy = os.path.join(
+                self.bundle_path(app), "app", "include",
+                "python{}m".format(self.python_version_tag),
```

Comment:
This path doesn't exist, is the trailing 'm' a mistake?

Reply:
The "m" is a suffix added by Python when it is configured with --with-pymalloc. It seems this is only valid for 3.6 and 3.7: https://github.com/beeware/Python-Android-support/blob/master/main.sh#L84 I'll fix this.